buildSrc/src/main/kotlin/published.gradle.kts

```diff
@@ -126,9 +125,6 @@ tasks.withType(PublishToMavenRepository::class.java).configureEach {
 tasks.withType(Jar::class.java).configureEach {
   notCompatibleWithConfigurationCache("")
 }
-tasks.withType(SourcesJar::class.java).configureEach {
-  notCompatibleWithConfigurationCache("")
-}
```

Comment:
The `SourcesJar` task was removed in the [latest release](https://github.com/vanniktech/gradle-maven-publish-plugin/pull/507) based on the PR description there _might_ be an equivalent task I could replace this with but I'm not entirely sure as I haven't worked with this plugin before.